server/src/main/java/io/pravega/schemaregistry/server/rest/ServiceConfig.java

```diff
@@ -40,7 +40,7 @@ private ServiceConfig(String host, int port, boolean tlsEnabled, String tlsCertF
         Exceptions.checkNotNullOrEmpty(host, "host");
         Exceptions.checkArgument(port > 0, "port", "Should be positive integer");
         Exceptions.checkArgument(!tlsEnabled || (!Strings.isNullOrEmpty(tlsCertFilePath) &&
-                        !Strings.isNullOrEmpty(serverKeyStoreFilePath)), "keyFilePath", 
+                        !Strings.isNullOrEmpty(serverKeyStoreFilePath) && !Strings.isNullOrEmpty(tlsKeyStorePasswordFilePath)), "keyFilePath", 
```

Comment:
These checks are unnecessarily complicated, although the intended condition being checked is very simple. Primarily, due to use of negative conditions. It should be simplified for easier reading.

Reply:
suggestions? the intent is as follows:
if tls is enabled then certificate and private key and private key's password should be present.